config/canonical.json

```diff
@@ -13901,10 +13907,12 @@
     "tags": {
       "amenity": "pharmacy",
       "brand": "くすりの福太郎",
+      "brand:en": "Kusurino FUKUTARO",
```

Comment:
Yes. Because, in Japanese Wikipedia is write as 
"Kusurino FUKUTARO" (with capital letters).

Reply:
(since both traditional [familyname givenname] and Western [givenname familyname] name orders are used in Japan now, the caps are sometimes used to remove ambiguity [where it exists; sometimes it is obvious] about which is the family name)